scripts/upload-apk.sh

```diff
@@ -36,6 +36,11 @@ git push origin apk --force --quiet > /dev/null
 
 if [[ $TRAVIS_BRANCH =~ ^(master)$ ]]; then
     cd ..
+
+    # Copy Screenshots
+    mkdir -p "fastlane/metadata/android/en-US/images/phoneScreenshots"
+    find docs -type f -name "screen-?.png" -exec cp {} "fastlane/metadata/android/en-US/images/phoneScreenshots/" \;
```

Comment:
Just use cp

Reply:
@iamareebjamal regex does not work in cp

Reply:
yeah not using regex, glob can work even in cp. Updated